packages/snap-preact/components/src/components/Organisms/BranchOverride/BranchOverride.tsx

```diff
@@ -277,16 +276,10 @@ export const BranchOverride = (properties: BranchOverrideProps): JSX.Element =>
 		setThemeName('failureTheme');
 	}
 
-	const styling: RootNodeProperties = { 'ss-name': props.name };
-	const stylingProps = props;
-
-	if (styleScript && !disableStyles) {
-		styling.css = [styleScript(stylingProps), style];
-	} else if (!disableStyles) {
-		styling.css = [CSS.override({ componentTheme: componentThemes[themeName as keyof typeof componentThemes], ...stylingProps }), style];
-	} else if (style) {
-		styling.css = [style];
-	}
+	const styling = mergeStyles<BranchOverrideProps & { componentTheme: BranchOverrideTheme }>(
```

Comment:
We should not be passing anything outside of BranchOverrideProps here. This may require a large refactor to this component outside of the scope. If you are able please refactor otherwise add a TODO above this line.